packages/shared/src/utils/contract/contract-info.ts

```diff
@@ -11,7 +11,7 @@ export const mockContractInfo = (obj: Partial<TContractInfo> = {}): TContractInf
         currency: 'USD',
         current_spot: 2415.18,
         current_spot_display_value: '2415.18',
-        current_spot_time: 1700481935,
+        current_spot_time: 1700481950,
```

Comment:
Increased the current spot time, because normally it shouldn't be less then date start (line 17). It was critical due to forward staring contracts as only they have such case with dates.